parsons/census/census.py

```diff
@@ -0,0 +1,64 @@
+import logging
+from parsons.utilities.api_connector import APIConnector
+from parsons.utilities import check_env
+from parsons import Table
+
+logger = logging.getLogger(__name__)
+
+
+class Census(object):
+    """
+    Class that creates a connector to the Census Bureau API
+    """
+
+    def __init__(self, api_key=None):
+        """
+        Instantiate Census class.
+
+           Args:
+                api_key: string, key for Census API access
+                 (optional, can also be pulled from environment variable CENSUS_API_KEY)
+        """
+        self.api_key = check_env.check("CENSUS_API_KEY", api_key)
+        self.host = "https://api.census.gov/data"
+
+    def get_census(
+        self, year=None, dataset_acronym=None, variables=None, location=None
+    ):
+        """
+        Pull census data. For background check out the Census API Guide at
+                https://www.census.gov/data/developers/guidance/api-user-guide.html
```

Comment:
Can you turn this into a link? You can use the same syntax in docstrings in code that you would in a .rst file